# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\yaml\reader.py
# hypothesis_version: 6.140.2

[4096, '\x00', '<byte string>', '<file>', '<unicode string>', 'Reader', 'ReaderError', '[^\t\n\r -~\x85\xa0-\ud7ff\ue000-ÔøΩêÄÄ-\U0010ffff]', 'name', 'strict', 'unicode', 'utf-16-be', 'utf-16-le', 'utf-8', '\ufeff']